docs/kratos/concepts/email-sms.md

```diff
@@ -394,3 +394,18 @@ courier:
 {{- end -}}
 
 ```
+
+## Handling errors
```

Comment:
@tomekpapiernik fyi this is currently open-source only as we limit the retries in cloud to exactly 5 and don't want to make it configurable.